src/networking.c

```diff
@@ -2783,9 +2852,10 @@ sds catClientInfoString(sds s, client *client) {
         replBufBlock *cur = listNodeValue(client->ref_repl_buf_node);
         used_blocks_of_repl_buf = last->id - cur->id + 1;
     }
+    sds meta = client->meta ? getAllClientMetaFields(client->meta) : sdsempty();
```

Comment:
```suggestion
    sds meta = client->meta ? getAllClientMetaFields(client->meta) : NULL;
```
sdsfree() will gracefully handle null values passed in. No need to allocate an empty object.